oasis/deployments.md

```diff
@@ -190,6 +194,48 @@ for how long passwords are valid. ArangoDB Oasis can automatically change the
 4. You can expand the __Root password__ panel to see when the password was
    rotated last. The rotation takes place every three months.
 
+## How to edit a deployment
+
+You can modify a deployment’s configuration, including the ArangoDB version
+that is being used, change the memory, CPU, and disk size, or even switch from
+a OneShard deployment to a Sharded one if your data set no longer fits in a
+single node. 
+
+{% hint 'info' %}
+To edit an existing deployment, you must have the necessary set of permissions
+attached to your role. Read more about [roles and permissions](access-control.html#roles).
+{% endhint %}
+
+1. Go to the **Projects** section and select an existing deployment from the list. 
+2. Open the deployment you want to change. 
+3. On the **Overview** tab, click the **Edit** button with an ellipsis (`…`)
+   icon. 
```

Comment:
not sure if we wan to mention here that if you don't have the rights to switch to a private-endpoint the Edit button will be available directly (no ellipsis will the there).